homeassistant/components/vicare/__init__.py

```diff
@@ -37,6 +38,35 @@
 _TOKEN_FILENAME = "vicare_token.save"
 
 
+async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+
+    if entry.version > 1:
```

Comment:
See the same integration I gave as example on how to test this.

Reply:
Should also migration to future minor version be prevented?

Reply:
Not needed as minor versions should not break anything in between.